[2022-10-07 13:24:26] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@619c93ca
[2022-10-07 13:24:26] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 13:29:05] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@5e5ddfbc
[2022-10-07 13:29:06] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 13:35:14] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@5e5ddfbc
[2022-10-07 13:35:14] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 13:39:17] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@619c93ca
[2022-10-07 13:39:18] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 13:49:12] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@4a216eb4
[2022-10-07 13:49:13] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 14:00:08] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@251d7fdd
[2022-10-07 14:00:08] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 14:30:26] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@31aab981
[2022-10-07 14:30:26] ERROR [main] ContextStartEventHandler.onApplicationEvent(26) | java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
java.sql.SQLSyntaxErrorException: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeLargeUpdate(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedStatement.executeUpdate(Unknown Source)
	at edu.hanoi.jazz.ContextStartEventHandler.creatTable(ContextStartEventHandler.java:37)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:22)
	at edu.hanoi.jazz.ContextStartEventHandler.onApplicationEvent(ContextStartEventHandler.java:14)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:378)
	at org.springframework.context.support.AbstractApplicationContext.start(AbstractApplicationContext.java:1432)
	at edu.hanoi.jazz.SpringJazzApplication.main(SpringJazzApplication.java:14)
Caused by: ERROR 42X01: Syntax error: Encountered "creat" at line 1, column 1.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)
	at org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)
	... 13 more
[2022-10-07 14:32:56] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@26be9a6
[2022-10-07 14:34:18] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@26be9a6
[2022-10-07 14:34:18] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 14:34:32] INFO [http-nio-8080-exec-8] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@3fb32b87
[2022-10-07 14:35:09] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@26be9a6
[2022-10-07 14:35:09] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 14:35:16] INFO [http-nio-8080-exec-2] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@78b0f4cb
[2022-10-07 14:35:16] INFO [http-nio-8080-exec-2] GroupDAOImpl.insert(22) | Save group truongtumeng done!
[2022-10-07 15:13:35] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@31aab981
[2022-10-07 15:13:35] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 15:13:56] INFO [http-nio-8080-exec-5] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@6fb6f8df
[2022-10-07 15:13:56] INFO [http-nio-8080-exec-5] GroupDAOImpl.insert(23) | Save group TieuMeMe done!
[2022-10-07 15:42:04] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@411fa0ce
[2022-10-07 15:42:04] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 15:42:20] INFO [http-nio-8080-exec-3] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@1c1e52d1
[2022-10-07 15:45:22] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@251d7fdd
[2022-10-07 15:45:22] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 15:45:35] INFO [http-nio-8080-exec-4] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@3f337ea1
[2022-10-07 15:48:13] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@2fd39436
[2022-10-07 15:48:13] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 15:48:23] INFO [http-nio-8080-exec-5] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@1c1e52d1
[2022-10-07 15:48:48] INFO [main] ContextStartEventHandler.onApplicationEvent(20) |  context start application org.springframework.jdbc.datasource.DriverManagerDataSource@411fa0ce
[2022-10-07 15:48:48] INFO [main] ContextStartEventHandler.creatTable(34) | Table HN_GROUP already exists!
[2022-10-07 15:49:00] INFO [http-nio-8080-exec-2] ContextStartEventHandler.addNew(36) | add new group ------ >edu.hanoi.jazz.dao.model.Group@2975d7e
[2022-10-07 15:49:00] INFO [http-nio-8080-exec-2] GroupDAOImpl.insert(24) | Save group Vnua life done!
